docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -292,7 +302,6 @@ module "cluster" {
 EOF
 terraform apply -target "module.cluster.module.lb.module.hiera"
 ----
-
```

Comment:
Is this blank line deletion intentional? Doesn't this break the next bullet point?

Reply:
oh damn, nice find! no, wasn't intentional.